uaclient/api/u/pro/security/fix/__init__.py

```diff
@@ -647,19 +833,27 @@ def _generate_fix_plan(
     if not src_pocket_pkgs:
         return fix_plan.fix_plan
 
-    all_already_installed = True
     for pocket in [
         UBUNTU_STANDARD_UPDATES_POCKET,
         UA_INFRA_POCKET,
         UA_APPS_POCKET,
     ]:
         pkg_src_group = src_pocket_pkgs[pocket]
         binary_pkgs = binary_pocket_pkgs[pocket]
+        source_pkgs = [src_pkg for src_pkg, _ in pkg_src_group]
+        pocket_name = get_pocket_short_name(pocket)
```

Comment:
Yes, this is totally feasible, but I think it is better to make that change on the PR that will retire the security module. My concern is that this PR is already too big as is and even though I believe this will be a straightforward refactor, I may be wrong and the complexity of this PR will just grow

Reply:
That sounds good to me +1